ethereum/core/src/main/java/org/hyperledger/besu/ethereum/worldstate/Pruner.java

```diff
@@ -100,7 +102,10 @@ private void mark(final BlockHeader header) {
   }
 
   private void sweep() {
-    LOG.info("Begin sweeping unused nodes for pruning. Retention period: {}", blocksRetained);
+    LOG.info(
```

Comment:
This should be at debug level for production.

Reply:
Agreed.